specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/networkInterface.json

```diff
@@ -1153,6 +1157,14 @@
           "readOnly": true,
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the network interface resource."
+        },
+        "nicType": {
+          "type": "string",
+          "description": "Type of Network Interface resource."
```

Comment:
Is it an enumable property? If so, you should set it in enum type.

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/5015df257c6f0f8f55e2491a5c13b6ac589c5288/specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/customIpPrefix.json#L366-L381